<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="AAD-W17Q1DRL3" failures="4" tests="5" name="Managed_API_Page" time="763.059" errors="0" timestamp="15 Dec 2022 18:09:35 GMT">
  <testcase classname="com.informatica.tests.APIGroupsTest" name="@BeforeTest init" time="93.029">
    <failure type="com.microsoft.playwright.TimeoutError" message="Timeout 60000ms exceeded">
      <![CDATA[com.microsoft.playwright.TimeoutError: Timeout 60000ms exceeded
at com.microsoft.playwright.impl.WaitableTimeout.get(WaitableTimeout.java:41)
at com.microsoft.playwright.impl.WaitableRace.get(WaitableRace.java:44)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:103)
at com.microsoft.playwright.impl.FrameImpl.waitForLoadStateImpl(FrameImpl.java:825)
at com.microsoft.playwright.impl.FrameImpl.waitForLoadStateImpl(FrameImpl.java:810)
at com.microsoft.playwright.impl.PageImpl.lambda$waitForLoadState$69(PageImpl.java:1260)
at com.microsoft.playwright.impl.WaitForEventLogger.get(WaitForEventLogger.java:46)
at com.microsoft.playwright.impl.ChannelOwner.withWaitLogging(ChannelOwner.java:72)
at com.microsoft.playwright.impl.PageImpl.waitForLoadState(PageImpl.java:1259)
at com.microsoft.playwright.Page.waitForLoadState(Page.java:6025)
at com.informatica.pages.LoginPage.ClickonAPIMChiklet(LoginPage.java:50)
at com.informatica.tests.APIGroupsTest.init(APIGroupsTest.java:39)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest init -->
  <testcase classname="com.informatica.tests.BaseTest" name="@AfterTest tearDown" time="1.222">
    <failure type="com.microsoft.playwright.PlaywrightException" message="Failed to read message from driver, pipe closed.">
      <![CDATA[com.microsoft.playwright.PlaywrightException: Failed to read message from driver, pipe closed.
at com.microsoft.playwright.impl.PipeTransport.poll(PipeTransport.java:75)
at com.microsoft.playwright.impl.Connection.processOneMessage(Connection.java:157)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:101)
at com.microsoft.playwright.impl.Connection.sendMessage(Connection.java:105)
at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:94)
at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:90)
at com.microsoft.playwright.impl.BrowserContextImpl.closeImpl(BrowserContextImpl.java:196)
at com.microsoft.playwright.impl.BrowserContextImpl.lambda$close$1(BrowserContextImpl.java:170)
at com.microsoft.playwright.impl.LoggingSupport.lambda$withLogging$0(LoggingSupport.java:36)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:47)
at com.microsoft.playwright.impl.ChannelOwner.withLogging(ChannelOwner.java:79)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:35)
at com.microsoft.playwright.impl.BrowserContextImpl.close(BrowserContextImpl.java:170)
at com.informatica.tests.BaseTest.tearDown(BaseTest.java:294)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.EOFException
at com.microsoft.playwright.impl.ReaderThread.readIntLE(PipeTransport.java:110)
at com.microsoft.playwright.impl.ReaderThread.readMessage(PipeTransport.java:139)
at com.microsoft.playwright.impl.ReaderThread.run(PipeTransport.java:125)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="com.informatica.tests.BaseTest" name="@AfterTest tearDown" time="0.344">
    <failure type="com.microsoft.playwright.PlaywrightException" message="Failed to read message from driver, pipe closed.">
      <![CDATA[com.microsoft.playwright.PlaywrightException: Failed to read message from driver, pipe closed.
at com.microsoft.playwright.impl.PipeTransport.poll(PipeTransport.java:75)
at com.microsoft.playwright.impl.Connection.processOneMessage(Connection.java:157)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:101)
at com.microsoft.playwright.impl.Connection.sendMessage(Connection.java:105)
at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:94)
at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:90)
at com.microsoft.playwright.impl.BrowserContextImpl.closeImpl(BrowserContextImpl.java:196)
at com.microsoft.playwright.impl.BrowserContextImpl.lambda$close$1(BrowserContextImpl.java:170)
at com.microsoft.playwright.impl.LoggingSupport.lambda$withLogging$0(LoggingSupport.java:36)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:47)
at com.microsoft.playwright.impl.ChannelOwner.withLogging(ChannelOwner.java:79)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:35)
at com.microsoft.playwright.impl.BrowserContextImpl.close(BrowserContextImpl.java:170)
at com.informatica.tests.BaseTest.tearDown(BaseTest.java:294)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.EOFException
at com.microsoft.playwright.impl.ReaderThread.readIntLE(PipeTransport.java:110)
at com.microsoft.playwright.impl.ReaderThread.readMessage(PipeTransport.java:139)
at com.microsoft.playwright.impl.ReaderThread.run(PipeTransport.java:125)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="com.informatica.tests.managedAPITest" name="accessAnonManagedAPI" time="60.108">
    <failure type="java.lang.AssertionError" message="Exception while validating AccessAnonManagedAPi">
      <![CDATA[java.lang.AssertionError: Exception while validating AccessAnonManagedAPi
at com.informatica.tests.managedAPITest.accessAnonManagedAPI(managedAPITest.java:121)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- accessAnonManagedAPI -->
  <testcase classname="com.informatica.tests.managedAPITest" name="accessBasicManagedAPI" time="76.641">
    <failure type="java.lang.AssertionError" message="Exception while validating AccessBasicManagedAPI">
      <![CDATA[java.lang.AssertionError: Exception while validating AccessBasicManagedAPI
at com.informatica.tests.managedAPITest.accessBasicManagedAPI(managedAPITest.java:158)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- accessBasicManagedAPI -->
  <testcase classname="com.informatica.tests.managedAPITest" name="validateWSDLURL" time="60.264">
    <failure type="java.lang.AssertionError" message="Exception while validating WSDL URL">
      <![CDATA[java.lang.AssertionError: Exception while validating WSDL URL
at com.informatica.tests.managedAPITest.validateWSDLURL(managedAPITest.java:199)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- validateWSDLURL -->
  <testcase classname="com.informatica.tests.managedAPITest" name="ClickOnEnablePrivacyPolicyWarning" time="261.46">
    <failure type="java.lang.AssertionError" message="Exception while clicking ClickOnEnablePrivacyPolicyWarningValidation">
      <![CDATA[java.lang.AssertionError: Exception while clicking ClickOnEnablePrivacyPolicyWarningValidation
at com.informatica.tests.managedAPITest.ClickOnEnablePrivacyPolicyWarning(managedAPITest.java:272)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- ClickOnEnablePrivacyPolicyWarning -->
  <testcase classname="com.informatica.tests.managedAPITest" name="ClickOnEnablePrivacyPolicyBlock" time="264.965"/>
  <testcase classname="com.informatica.tests.managedAPITest" name="ManagedAPINameToolTip" time="0.0">
    <ignored/>
  </testcase> <!-- ManagedAPINameToolTip -->
</testsuite> <!-- Managed_API_Page -->
